startup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+cp system_can.dbc /
```

Comment:
It would be nice to not have to move system_can.dbc to the root directory. Instead, it might be better for a subshell to be created that would run the python script from /home/pi/box/telemetry_xiv/telemetry_scripts